jujugui/static/gui/src/app/assets/images/non-sprites/tick.svg

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   viewBox="0 0 10 8"
+   version="1.1"
+   id="svg5918"
+   height="8"
+   width="10">
+  <defs
+     id="defs5920">
+    <clipPath
+       clipPathUnits="userSpaceOnUse"
+       id="clipPath16">
+      <path
+         d="m 0,595.28 841.89,0 L 841.89,0 0,0 0,595.28 Z"
+         id="path18" />
+    </clipPath>
+    <clipPath
+       clipPathUnits="userSpaceOnUse"
+       id="clipPath16-0">
+      <path
+         d="m 0,595.28 841.89,0 L 841.89,0 0,0 0,595.28 Z"
+         id="path18-1" />
+    </clipPath>
+  </defs>
+  <metadata
+     id="metadata5923">
+    <rdf:RDF>
+      <cc:Work
+         rdf:about="">
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title></dc:title>
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+  <g
+     transform="translate(-182.93849,-568.7201)"
+     id="layer1">
+    <g
+       id="g5687"
+       transform="matrix(0.66666607,0,0,0.66666668,-633.06096,6.8119427)"
+       style="display:inline" />
+    <g
+       transform="translate(-769.06151,404.35792)"
+       id="g5346"
+       style="display:inline">
+      <g
+         transform="matrix(0.33333333,0,0,0.33333333,942.33334,88.241445)"
+         id="g6395"
+         style="display:inline">
+        <rect
+           style="opacity:0.21171169;fill:none;stroke:none"
+           id="rect6397"
+           width="24"
+           height="24"
+           x="31.999992"
+           y="228.3622" />
+        <path
+           style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:15px;line-height:125%;font-family:Ubuntu;-inkscape-font-specification:Ubuntu;text-align:center;letter-spacing:0px;word-spacing:0px;writing-mode:lr-tb;text-anchor:middle;display:inline;fill:#ffffff;fill-opacity:1;stroke:none"
+           d="m 54.157606,231.70402 -0.173484,0.15201 -12.466718,10.91973 -7.308154,-6.26905 -2.20926,2.49843 9.519224,10.01524 14.480776,-15.23796 -1.842384,-2.07821 z"
+           id="path6399" />
+      </g>
+    </g>
+  </g>
+</svg>
```

Comment:
Looks like SVGO borked the optimisation. Let's go with this over and I'll try and get a less complicated one from design.